modules/plot/main/coffee/Graph.coffee

```diff
@@ -294,15 +297,16 @@ class Graph extends hx.EventEmitter
       .attr('width', @plotArea.x2 - @plotArea.x1)
       .attr('height', @plotArea.y2 - @plotArea.y1)
 
+    @emit 'render'
```

Comment:
This event was added mostly to facilitate testing, but you might want to document it anyway.